app/models/client.rb

```diff
@@ -96,8 +96,8 @@ def self.sortable_intake_attributes
   scope :with_eager_loaded_associations, -> { includes(:vita_partner, :intake, :tax_returns, tax_returns: [:assigned_user]) }
   scope :sla_tracked, -> { distinct.joins(:tax_returns).where.not(tax_returns: { status: TaxReturnStateMachine::EXCLUDED_FROM_SLA }) }
 
-  scope :first_unanswered_incoming_interaction_communication_breaches, ->(breach_threshold_datetime) do
-    sla_tracked.where(arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_datetime))
+  scope :first_unanswered_incoming_interaction_between, ->(range) do
```

Comment:
Given the relative simplicity of the scope, I think it's better to remove the scope entirely, and have the callsites directly query `Client.sla_tracked.where(first_unanswered_incoming_interaction_at: ...@breach_date)`. Bold suggestion, but that's what comes to mind. Not a blocker; I can live with the scope existing.

Reply:
its called in multiple places, and it's important that the implementations are the same.